coverage/config.py

```diff
@@ -227,6 +227,9 @@ def __init__(self):
         self.json_pretty_print = False
         self.json_show_contexts = False
 
+        # Defaults for [lcov]
```

Comment:
This comment is a little misleading, since there is no parsing of an `[lcov]` section in the config file.  Do you want to support setting the lcov output file in the config file?